### DISCUSSION TITLE
Style Radio label?

### DISCUSSION THREAD

**Discussion by benasher44** [chakra-ui/chakra-ui#3001] [Q&A] [answered]
Hi! I'm trying to style a Radio's internal label. Specifically, I want to set "word-wrap: unset" to allow long text to break anywhere. Is there a way to access/configure a Radio's label's props?

**Comment by tioluwani94** [comment, marked as answer]
Hi @benasher44, yes you can access/configure a Radio's label style by extending the Radio component styles.

Here's an example

```
const theme = extendTheme({
  components: {
    Radio: {
      parts: ['label'],
      baseStyle: {
        label: {
          wordWrap: 'unset',
        },
      },
    },
  },
});

function MyApp({ Component, pageProps }) {
  return (
    <ChakraProvider theme={theme}>
        <App />
    </ChakraProvider>
  );
}

export default MyApp;
```

You can also check the docs for more information:
https://chakra-ui.com/docs/theming/customize-theme
https://chakra-ui.com/docs/theming/component-style

**Reply by benasher44** [reply]
Ah neat thanks!

**Comment by israelperez** [comment]
Multi part components are to well explained in the documentation. 
if you leave out parts: ['label'], it still works? what is this line actually doing?
